website/source/api/system/generate-root.html.md

```diff
@@ -38,16 +38,24 @@ $ curl \
   "required": 3,
   "encoded_token": "",
   "pgp_fingerprint": "",
+  "otp_length": 24,
   "complete": false
 }
 ```
 
 If a root generation is started, `progress` is how many unseal keys have been
 provided for this generation attempt, where `required` must be reached to
 complete. The `nonce` for the current attempt and whether the attempt is
-complete is also displayed. If a PGP key is being used to encrypt the final root
-token, its fingerprint will be returned. Note that if an OTP is being used to
-encode the final root token, it will never be returned.
+complete is also displayed.
+
+If a PGP key is being used to encrypt the final root
+token, its fingerprint will be returned.
```

Comment:
// , This seems like it would be intuitive, but I guess it doesn't hurt to point it out?